python/aitemplate/backend/cuda/gemm_universal/common.py

```diff
@@ -222,7 +222,7 @@
 
 {{indent}}};
 {% if is_profiler %}
-{{indent}}// https://www.youtube.com/watch?v=rRwxfYlgG-M
+{{indent}}// Namewee - China REGGAETON: https://www.youtube.com/watch?v=rRwxfYlgG-M
```

Comment:
Lol. Hope you enjoy the music! You discovered an Easter egg! Maybe let's just remove this line to avoid complications. :)

Reply:
ok, updated the PR.